### PR TITLE
WatchFaceG7710: Simplify time update check

### PR DESCRIPTION
**Pull request by Riksu9000**

### CODE REVIEW

File: src/displayapp/screens/WatchFaceCasioStyleG7710.cpp [hunk 1]
```diff
@@ -222,43 +222,36 @@ void WatchFaceCasioStyleG7710::Refresh() {
     lv_label_set_text_static(notificationIcon, NotificationIcon::GetIcon(notificationState.Get()));
   }
 
-  currentDateTime = dateTimeController.CurrentDateTime();
-
+  currentDateTime = std::chrono::time_point_cast<std::chrono::minutes>(dateTimeController.CurrentDateTime());
   if (currentDateTime.IsUpdated()) {
     auto hour = dateTimeController.Hours();
     auto minute = dateTimeController.Minutes();
```
Comment (FintasticMan) [inline src/displayapp/screens/WatchFaceCasioStyleG7710.cpp:228]: Don't use auto for integers.

File: src/displayapp/screens/WatchFaceCasioStyleG7710.cpp [hunk 2]
```diff
@@ -222,43 +222,36 @@ void WatchFaceCasioStyleG7710::Refresh() {
...
+
+    currentDate = std::chrono::time_point_cast<days>(currentDateTime.Get());
+    if (currentDate.IsUpdated()) {
+      auto weekNumberFormat = "%V";
 
-    if ((year != currentYear) || (month != currentMonth) || (dayOfWeek != currentDayOfWeek) || (day != currentDay)) {
+      auto year = dateTimeController.Year();
+      auto month = dateTimeController.Month();
+      auto day = dateTimeController.Day();
+      auto dayOfYear = dateTimeController.DayOfYear();
```
Comment (FintasticMan) [inline src/displayapp/screens/WatchFaceCasioStyleG7710.cpp:254]: Don't use auto for the integers here.

**Review by FintasticMan — CHANGES_REQUESTED** [review]
Except for the uses of auto, this looks good to me.

File: src/displayapp/screens/WatchFaceCasioStyleG7710.cpp [hunk 3]
```diff
@@ -222,43 +222,36 @@ void WatchFaceCasioStyleG7710::Refresh() {
...
+      lv_label_set_text(label_time_ampm, ampmChar);
+      lv_label_set_text_fmt(label_time, "%2d:%02d", hour, minute);
+    } else {
+      lv_label_set_text_fmt(label_time, "%02d:%02d", hour, minute);
     }
+    lv_obj_realign(label_time);
+
+    currentDate = std::chrono::time_point_cast<days>(currentDateTime.Get());
+    if (currentDate.IsUpdated()) {
+      auto weekNumberFormat = "%V";
```
Comment (FintasticMan) [inline src/displayapp/screens/WatchFaceCasioStyleG7710.cpp:249]: It's better to not use auto for arrays IMO, it just makes it confusing.

**Review by JF002 — APPROVED** [review]

**Pull request merged**